content/blog/gatsby-blog/index.md

```diff
@@ -0,0 +1,323 @@
+
+```bash
+# Install Gatsby cli
+npm install -g gatsby-cli
+
+# Create a new gatsby site using the starter-blog starter
+gatsby new my-blog https://github.com/gatsbyjs/gatsby-starter-blog
+
+```
+
+Check the output to see how to run the app but at this point you have a working blog.
+
+These are the most relevant files:
+
+- `gatsby-config.js` to edit data about your site such as author, description, titles, etc
+- `gatsby-config.js` to add your Google Analytics id
+- `content/name-of-the-post/index.md` to see how to write posts in markdown
+- add `import "prismjs/themes/prism-tomorrow.css"` to `gatsby-browser.js` to enable **syntax highlighting** of code blocks. This starter uses [prism.js](https://prismjs.com/) but it does not add the css theme by default. Check prism.js documentation to see other themes available.
+
+
+And you are done! Told you! it is super fast and easy.
+
+
+
+Useful links
+- https://www.gatsbyjs.org/docs/quick-start/
+
+
+
+## Host your site in [Github Pages](https://pages.github.com/)
+
+Github pages is a static page hosting solution, if you are a software developer
+this hosting solution is pretty near to our daily roaming places so that is why I choose it.
+
+There are tons of alternatives but if you already host your page's code in github
+then this will be pretty easy to setup.
+
+Follow the instructions on [the website](https://pages.github.com/) to enable github pages.
+
+These are the extra steps I took for this blog
+
+### 1 make `develop` your default branch (only for _project site_)
+
+Since this blog qualifies as a *project site* (per repo) instead of a *user or organization site*
+we need to have the following repo branch configuration:
+
+- `develop`: where our gatsby code lives
+- `master`: where our gatsby-generated static files for our website are stored.
+
+These generated static files will be the ones that github pages will deploy and show.
+
+Have in mind that you will be managing two code bases, one is your gatsby site, the code that you will
+be interacting with as a developer and then we have the generated static site code that will be the one
+that we serve.
+Think of these two code bases as the source code and the compiled target code in traditional
+compiled programming languages such as C/C++ or Rust.
+
+### 2 Adjust the `package.json` deploy script
+
+It should look like this
+
+```json
+"deploy": "gatsby build --prefix-paths && gh-pages -d public -b master"
+```
+
+- `--prefix-paths` is only useful if you are going to use the default github domain: `username.github.io/reponame`. Remove it if you are making your site accessible from some `domain.com`
+- `gh-pages` is a helper node package that you need to install: `npm install --dev gh-pages`
+- `-b master` specifies to `gh-pages` package that you want to deploy to the `master` branch
+
+
+### 3 Adjust your `pathPrefix`
+
+IMPORTANT! only do this if you are using the default github domain. If you are using a custom domain,
```

Comment:
I would capitalize the first letter of "only do this..." mainly for consistency (and I think it looks better as well), since in previous occurrences you did that.

Reply:
This happens below on "your site is live" as well.